# Subscriber
car_state_topic_name: /estimation/slam/state

map_topic_name: /map

# Publisher
path_generate_topic_name: /planning/ref_path

ref_path_topic_name: /visual/ref_path

# Trajectory params:
desire_vel: 15

car_length: 1.88

look_ahead: 20

# other params:
N: 40

dt: 0.04

initial_velocity: 2

mission: trackdrive          # [trackdrive, skidpad, acceleration]

simulation: true

node_rate: 100   # [Herz]
